demisto_sdk/commands/format/tests/test_formatting_yml_test.py

```diff
@@ -1868,3 +1869,27 @@ def test_yml_run_format_exception_handling(format_object, mocker):
         logger_info.call_args_list,
         "Failed to update file my_file_path. Error: MY ERROR",
     )
+
+
+def test_handle_hidden_marketplace_params():
+    """
+    Given
+    - Integration yml with parameters configured with Hidden: False.
+    When
+    - Running the handle_hidden_marketplace_params function.
+    Then
+    - Ensures the hidden value is equivalent to master branch.
+    """
+    base_yml = IntegrationYMLFormat(SOURCE_FORMAT_INTEGRATION_VALID, path="schema_path")
+    with open(SOURCE_FORMAT_INTEGRATION_VALID_OLD_FILE) as old_yml_file:
+        base_yml.old_file = yaml.load(old_yml_file)
```

Comment:
```suggestion
    base_yml.old_file = get_yaml(SOURCE_FORMAT_INTEGRATION_VALID_OLD_FILE)
```
may have to import it